src/android/io/branch/BranchSDK.java

```diff
@@ -46,12 +46,12 @@ public BranchSDK()
     {
         this.activity = null;
         this.instance = null;
-        this.branchObj = null;
         this.callbackContext = null;
         this.onShareLinkDialogLaunched = null;
```

Comment:
From the current Readme.md:

"branchUniversalObj.onShareSheetLaunched(function () {
  console.log('Share sheet launched');
});"

so I believe that these on* callback handlers should be part of the branch object we're storing in our arraylist below; otherwise they'll get overridden when you create a second listener on a second branch universal object.